src/components/home/hero_section.js

```diff
@@ -1,52 +1,73 @@
 import React from "react"
 import PropTypes from "prop-types"
-import Img from "gatsby-image"
 import { StaticQuery, graphql } from "gatsby"
-import { ParallaxProvider, ParallaxBanner } from "react-scroll-parallax"
+import { ParallaxProvider, Parallax } from "react-scroll-parallax"
 
 import CallToAction from "../call_to_action"
-import LoadPlaceholder from "../load_placeholder"
+import ImageLoader from "../image_loader"
 import Text from "../text"
 import Title from "./hero_section/title"
+import useWindowSize from "../../utils/use_window_size"
 
 import styles from "./hero_section.module.css"
 
+const absoluteStyle = {
+  position: "absolute",
+  top: 0,
+  right: 0,
+  bottom: 0,
+  left: 0,
+}
+
 const ParallaxImage = ({
   baseDelay,
+  alt,
   mobileImage,
   desktopImage,
-  parallaxAmount,
-}) => (
-  <ParallaxBanner
-    layers={[
-      {
-        children: (
-          <LoadPlaceholder delay={baseDelay}>
-            {onLoad => (
-              <Img
-                fadeIn={false}
-                onLoad={onLoad}
-                fluid={[
-                  {
-                    ...mobileImage,
-                    media: "(max-width: 399px)",
-                  },
-                  {
-                    ...desktopImage,
-                    media: "(min-width: 400px)",
-                  },
-                ]}
-                imgStyle={{ display: "block" }}
-                style={{ height: "100%" }}
-              />
-            )}
-          </LoadPlaceholder>
-        ),
-        amount: parallaxAmount,
-      },
-    ]}
-  />
-)
+  objectPosition,
+  mobileTopMarign,
+  mobileBottomMarign,
+  desktopTopMarign,
+  desktoBottomMarign,
+}) => {
+  const size = useWindowSize()
```

Comment:
Why not destruct the variable to have `width`? I might be missing something but looks like it isn't even using the height.

Reply:
It's from https://usehooks.com/useWindowSize/. I would rather keep it as it is because it's general purpose and that way I can just copy it over again in case there's a bug and the code gets updated.